org.eclipse.jdt.core/search/org/eclipse/jdt/internal/core/search/indexing/DOMToIndexVisitor.java

```diff
@@ -0,0 +1,371 @@
+/*******************************************************************************
```

Comment:
The title of this PR is a bit misleading.  It implies you are just adding a new flag but you are adding this new code and this should be spelled out in the PR title and commit comment (e.g. Convert DOM to Index based on new flag setting)

Reply:
I fixed it.